README.md

```diff
@@ -11,6 +11,7 @@
 
 1. __Set up the environment.__
     - Put your __config__ in `/opt/aeacus/scoring.conf` or`C:\aeacus\scoring.conf`.
+    - Run `./aeacus configure` to configure Vulns, Only for Linux :( 
```

Comment:
(v)ulns, (o)nly and maybe remove the `:(`

Reply:
- Run `./aeacus configure` to launch GUI tool for configuring vulnerabilities

Reply:
Done